akka-http/src/main/scala/scala/util/matching/JavaConverters.scala

```diff
@@ -0,0 +1,10 @@
+package scala.util.matching
```

Comment:
Actually, package-private in Scala means public in Java; maybe I should just write the regex wrapper in Java, calling into `new Regex(thePattern, EMPTY_SEQ)`, and then use that in the directive? Seems a bit cleaner than reflection, and at least we have _one_ compiler future-checking it.